website/www/site/assets/scss/_playground.sass

```diff
@@ -17,6 +17,14 @@
 
 @import "media"
 
+.code-snippet-playground
+  padding: 0
+  background: none!important
```

Comment:
I'm assuming that we need to use `!important` here -- better to avoid it if we can.

Reply:
It overrides this `!important`: https://github.com/apache/beam/blob/acd0134ec61151336ccd49ff8a58b4d3675def6d/website/www/site/assets/scss/_global.sass#L112

Which in turn overrides this grey background for snippets introduced 6 years ago:
https://github.com/apache/beam/blob/acd0134ec61151336ccd49ff8a58b4d3675def6d/website/www/site/assets/scss/_syntax-highlighting.scss#L22

I removed that and all `!important`s that grew from it and see no difference, thank you.